### PR TITLE
vendor: update pd-client

### PR DESCRIPTION
**Pull request by nolouch**
PTAL @coocood @disksing

### CODE REVIEW

File: _vendor/src/github.com/gorilla/context/doc.go [hunk 1]
```diff
@@ -5,6 +5,12 @@
 /*
 Package context stores values shared during a request lifetime.
 
+Note: gorilla/context, having been born well before `context.Context` existed,
```
Comment (shenli) [inline _vendor/src/github.com/gorilla/context/doc.go]: Why update this repo?

Comment (nolouch) [inline _vendor/src/github.com/gorilla/context/doc.go]: PD glide let this repo up to date

Comment (siddontang) [inline _vendor/src/github.com/gorilla/context/doc.go]: seem that you change the commit version explicitly? glide couldn't update it automatically.

Comment (nolouch) [inline _vendor/src/github.com/gorilla/context/doc.go]: see in https://github.com/pingcap/pd/blob/master/glide.yaml#L97, it updates to the latest version automatically.

Comment (siddontang) [inline _vendor/src/github.com/gorilla/context/doc.go]: I think we should not use `~1.1.0`, but use a special commit version in glide.yaml.

Comment (nolouch) [inline _vendor/src/github.com/gorilla/context/doc.go]: yep

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@nolouch @disksing We could bench tidb again to see if there is any improvement.

**Pull request merged**